### DISCUSSION TITLE
Plans to remove `max_post_count`

### DISCUSSION THREAD

**Discussion by ruchernchong** [gautamkrishnar/blog-post-workflow#200] [Q&A] [answered]
Currently, we need to set a number for the `max_post_count` option.

By removing the `max_post_count` options, we are able to allow any amount of posts rather than arbitrarily set a huge number.

Are there any plans for this?

**Comment by gautamkrishnar** [comment, marked as answer]
@ruchernchong max_post_count allows you to customize the number of items you want to show on your readme. max_post_count is optional key. By default, it is 5. I do not have any plans to remove this count since it will break the readme for thousands of users using this project.

If you don't want that limit on your readme you can set it to any higher limit such as 9999.

**Reply by ruchernchong** [reply]
> @ruchernchong max_post_count allows you to customize the number of items you want to show on your readme. max_post_count is optional key. By default, it is 5. I do not have any plans to remove this count since it will break the readme for thousands of users using this project.
> 
> If you don't want that limit on your readme you can set it to any higher limit such as 9999.

Yeah, I was thinking of the `max_post_count` default value can be unlimited. But thanks for the response. I will just go ahead and set a large number as intended.
